packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesModule.kt

```diff
@@ -67,13 +65,9 @@ class UpdatesModule : Module() {
               localAssets[asset.key!!] = localAssetFiles[asset]!!
             }
           }
-          constants["localAssets"] = localAssets
+          this["localAssets"] = localAssets
         }
-      } catch (e: Exception) {
-        // do nothing; this is expected in a development client
```

Comment:
this comment was incorrect as it seems to also have applied to normal debug builds